### PR TITLE
fix: Course updates - wrong datepicker value

### PR DESCRIPTION
**Pull request by vladislavkeblysh**
- Add checking date in form

[2u-631](https://youtrack.raccoongang.com/issue/2U-631)

### CODE REVIEW

File: src/course-updates/update-form/UpdateForm.jsx [hunk 1]
```diff
@@ -73,7 +72,7 @@ const UpdateForm = ({
                   <DatePicker
                     name="date"
```
Comment (ruzniaievdm) [inline src/course-updates/update-form/UpdateForm.jsx:73]: maybe it would be better to add validation to schema?

Comment (vladislavkeblysh) [reply src/course-updates/update-form/UpdateForm.jsx:74]: Done

File: src/course-updates/update-form/UpdateForm.jsx [hunk 2]
```diff
@@ -17,12 +17,11 @@ import {
   convertToDateFromString,
   isValidDate,
 } from '../../utils';
-
-import messages from './messages';
-import { REQUEST_TYPES } from '../constants';
 import { DATE_FORMAT, DEFAULT_EMPTY_WYSIWYG_VALUE } from '../../constants';
+import { REQUEST_TYPES } from '../constants';
 import { WysiwygEditor } from '../../generic/WysiwygEditor';
```
Comment (PKulkoRaccoonGang) [inline src/course-updates/update-form/UpdateForm.jsx:22]:
```suggestion
import { WysiwygEditor } from '../../generic/WysiwygEditor';
import { REQUEST_TYPES } from '../constants';
```

**Review by PKulkoRaccoonGang — APPROVED** [review]

File: src/course-updates/update-form/UpdateForm.jsx (hunk 2 above)
Comment (vladislavkeblysh) [reply src/course-updates/update-form/UpdateForm.jsx]: Done

**Review by ruzniaievdm — APPROVED** [review]

**Pull request merged**